src/routes/kycRoutes.ts

```
@@ -5,6 +5,10 @@ import {
   requestDownloadUrl,
   scanWebhook,
 } from "../controllers/kycController";
+import {
+  createReward,
+  listRewards,
+} from "../controllers/kycValidatorRewardController";
```

ESLint

[error] Replace `⏎··createReward,⏎··listRewards,⏎` with `·createReward,·listRewards·`

(prettier/prettier)